ironfish/src/blockchain/blockchain.ts

```diff
@@ -126,6 +126,10 @@ export class Blockchain {
   onDisconnectBlock = new Event<[block: Block, tx?: IDatabaseTransaction]>()
   // When ever a block is added to a fork
   onForkBlock = new Event<[block: Block, tx?: IDatabaseTransaction]>()
+  // When ever the blockchain is reorganized
+  onReorganizeChain = new Event<
```

Comment:
Since this is on the blockchain we can remove the chain suffix. I think `blockchain.onReorganize` is obvious that the blockchain is reorganizing.